static-checks/rpmbuild-ctest/test.py

```diff
@@ -27,14 +27,42 @@
     ]
     util.subprocess_run(cmd, cwd=build_dir)
 
-    with open(build_dir / 'ctest_results') as f:
-        # Result format: X/Y Test  #X: test_name .................  test_result   Z sec
-        result_regex = re.compile(r'\d+\s+Test\s+#\d+:\s+([^\s]+)\s+\.+')
+    with open(build_dir / 'Testing' / 'Temporary' / 'LastTest.log') as f:
+        state = 'start'
+        test_output = []
+
         for line in f:
-            result_match = result_regex.search(line)
-            if result_match:
-                test_name = result_match.group(1)
-                result = 'pass' if 'Passed' in line else 'fail'
-                results.report(result, test_name)
+            line = line.rstrip('\n')
```

Comment:
It seems you do `test_output.append(line)` in every single state, maybe put it here instead?
```suggestion
            line = line.rstrip('\n')
            test_output.append(line)
```

Reply:
Only lines in the expected states are appended because otherwise we would get some overlaps between tests (there are some additional lines we don't care about).